src/uwtools/resources/jsonschema/chgres-cube.jsonschema

```diff
@@ -0,0 +1,205 @@
+                    },
+                    "lai_from_climo": {
+                      "type": "boolean"
+                    },
+                    "minmax_vgfrc_from_climo": {
+                      "type": "boolean"
+                    },
+                    "mosaic_file_input_grid": {
+                      "type": "string"
+                    },
+                    "mosaic_file_target_grid": {
+                      "type": "string"
+                    },
+                    "nsoill_out": {
+                      "type": "integer"
+                    },
+                    "nst_files_input_grid": {
+                      "type": "string"
+                    },
+                    "orog_dir_input_grid": {
+                      "type": "string"
+                    },
+                    "orog_dir_target_grid": {
+                      "type": "string"
+                    },
+                    "orog_files_input_grid": {
+                      "type": "string"
+                    },
+                    "orog_files_target_grid": {
+                      "type": "string"
+                    },
+                    "regional": {
+                      "type": "integer"
+                    },
+                    "sfc_files_input_grid": {
+                      "type": "string"
+                    },
+                    "sotyp_from_climo": {
+                      "type": "boolean"
+                    },
+                    "tg3_from_soil": {
+                      "type": "boolean"
+                    },
+                    "thomp_mp_climo_file": {
+                      "type": "string"
+                    },
+                    "tracers": {
+                      "type": "array"
+                    },
+                    "tracers_input": {
+                      "type": "array"
+                    },
+                    "varmap_file": {
+                      "type": "string"
+                    },
+                    "vcoord_file_target_grid": {
+                      "type": "string"
+                    },
+                    "vgfrc_from_climo": {
+                      "type": "boolean"
+                    },
+                    "vgtyp_from_climo": {
+                      "type": "boolean"
+                    },
+                    "wam_cold_start": {
+                      "type": "boolean"
+                    },
+                    "wam_parm_file": {
+                      "type": "string"
+                    }
+                  },
+                  "type": "object"
+                }
+              },
+              "required": [
+                "config"
+              ],
+              "type": "object"
+            }
+          },
+          "type": "object"
+        },
+        "run_dir": {
+          "type": "string"
+        }
+      },
+      "required": [
+        "execution",
+        "run_dir"
+      ],
+      "type": "object"
+    }
+  },
+  "type": "object"
+}
```

Comment:
I'd still very much like to see at least a minimal (perhaps following the one for `sfc-climo-gen.jsonschema`) test or set of tests for this schema, whether in this PR or in a more-or-less-immediate follow-on.

Reply:
Yep, I'm going to just do both schema tests as a follow-on